xgcm/grid.py

```diff
@@ -670,6 +678,7 @@ def min(
             fill_value,
             boundary_discontinuity,
             vector_partner,
+            keep_coords,
         )
 
```

Comment:
This is missing `@docstrings.dedent`, could you add it? Sorry for this. That must have gone missing some time ago...

Reply:
you mean for `max`, correct?

Reply:
Yip, sorry. The unfolding in github is quite annoying...